app/services/proofing/state_id_result.rb

```diff
@@ -56,6 +59,10 @@ def mva_exception?
       mva_unavailable? || mva_system_error? || mva_timeout?
     end
 
+    def jurisdiction_in_maintenance_window?
+      jurisdiction_in_maintenance_window
+    end
```

Comment:
This follows the pattern, but also feels... superfluous? Opinions on just passing through the instance variable rather than adding a predicate method?

Reply:
It's more common to not define the `attr_reader` for booleans and just have the predicat method like this

so more like:
```
    def jurisdiction_in_maintenance_window?
      !!@jurisdiction_in_maintenance_window
    end
```

Reply:
That was pretty much my instinct, but I followed the pattern of `success` (lines 38-40) that set up an `attr_reader` and then have a predicate method that just directly returns the `attr_reader` value.

(I think my other instinct was to write none of this code and just make line 77 be `jurisdiction_in_maintenance_window: !!@jurisidiction_in_maintenance_window`.)

Reply:
I've also used `alias_method` for this, though I get the impression that opinions are mixed w.r.t. aliasing.

https://github.com/18F/identity-idp/blob/ea73d99ef2b3f8cf501bb851eeda6db0e5d02b21/app/presenters/session_timeout_modal_presenter.rb#L18-L19